### DISCUSSION TITLE
Mac系统应该选择哪个版本下载安装

### DISCUSSION THREAD

**Discussion by banduoba** [AlistGo/alist#5003] [Q&A] [answered]
如题

**Comment by anwen-anyi** [comment, marked as answer]
开头的这俩就是 Mac用的，amd是Intel芯片，arm是M系列芯片 应该是这样的
![image](https://github.com/alist-org/alist/assets/56105412/3dbeffd7-9d5b-4260-90a4-a8cc55a8b62d)

**Reply by zhuziyi1989** [reply]
alist-darwin-amd64.tar.gz
alist-darwin-arm64.tar.gz

1. 此处的 darwin：Darwin由苹果公司于2000年所发布的一个开放源代码操作系统。Darwin是macOS和iOS操作环境的操作系统部分。苹果公司于2000年把Darwin发布给开放源代码社群。
2. 此处的 amd：基于 Intel 的 x86 架构处理器
3. 此处的 arm：基于 ARM 架构的 Apple Silicon 芯片（M 系列）

### alist macOS系统应该选择哪个版本？
### alist 苹果电脑苹果系统应该选择哪个版本？
### alist MacBook应该选择哪个版本？
### alist Mac系统应该选择哪个版本？

**Reply by anwen-anyi** [reply]
? 就这俩架构，你什么芯片就用什么版本呗

**Comment by banduoba** [comment]
OK 对的 一解决 谢谢
